packages/nuxt/src/server/sdk.ts

```diff
@@ -15,5 +15,29 @@ export function init(options: SentryNuxtOptions): Client | undefined {
 
   applySdkMetadata(sentryOptions, 'nuxt', ['nuxt', 'node']);
 
-  return initNode(sentryOptions);
+  const client = initNode(sentryOptions);
+
+  getGlobalScope().addEventProcessor(
+    Object.assign(
+      (event => {
+        if (event.type === 'transaction') {
+          // Filter out transactions for Nuxt build assets
+          // This regex matches the default path to the nuxt-generated build assets (`_nuxt`).
+          if (event.transaction?.match(/^GET \/_nuxt\//)) {
```

Comment:
l: Maybe let's add a small TODO comment here to dynamically match this path based on the nuxt build output folder so that we don't forget.